queries/ne.jinja2

```diff
@@ -32,7 +32,9 @@ SELECT
 {% endif %}
     'min_zoom', mz_boundary_min_zoom,
     'featurecla', featurecla,
-    'scalerank', scalerank
+    'scalerank', scalerank,
+    'ne_id', ne_id,
```

Comment:
Should ne_id and BRK_A3 be prefixed with `tz_` since they should only be used for internal processing?

Reply:
>     'ne_id', tz_ne_id,

Reply:
Nevermind... this is the DB ETL part, the `tz_` part happens in the query step.